### PR TITLE
Extract kpi module from OCA7management-system

### PR DESCRIPTION
**Pull request by dreispt**
This only extract the commit history.
The module is not installable, and a later PR doing the actual 9.0 port work is expected to follow.

### CODE REVIEW

**Comment by max3903** [comment]
@hsoft's contribution was covered by @savoirfairelinux ECLA.

File: mgmtsystem_kpi/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    "website": "http://www.savoirfairelinux.com",
+    "license": "AGPL-3",
+    "category": "Management System",
+    "complexity": "normal",
+    "description": """\
+This module provides the basis for creating key performance indicators,
+including static and dynamic thresholds (SQL query or Python code),
+on local and remote data sources.
+
+The module also provides the mecanism to update KPIs automatically.
```
Comment (atchuthan) [inline mgmtsystem_kpi/__openerp__.py:34]: h/mecanism/mechanism

Comment (dreispt) [inline mgmtsystem_kpi/__openerp__.py:34]: The module is inactive and is yet to be ported.
This is just a technical PR to move the history to this repo and allow a later PR with the port.

Comment (atchuthan) [inline mgmtsystem_kpi/__openerp__.py:34]: ok

**Pull request merged**